src/features/snackbar/index.js

```diff
@@ -74,7 +80,10 @@ class Snackbar extends Component {
             this.setState({
                 show: `NEW ITEM: ${itemReceived.split('-')[0]}`,
                 item: item,
+                equip: false,
             });
+            if (item.type !== 'potion' && item.name !== 'Rusty Sword')
+                this.setState({ equip: true });
```

Comment:
I think this might look nicer if combined into a single setState (equip: item.type !== 'potion' && item.name !== 'Rusty Sword').